README.md

```diff
@@ -272,6 +272,8 @@ Take into account the ``instrumentationTestTask`` could be different if you use
 
 Shot generates an HTML report you can review at the end of the recording or verification build. However, if you are running Shot in a CI environment which does not support saving the reporting files generated by the build you can verify your tests using this command ``./gradlew debugExecuteScreenshotTests -PprintBase64``. This will change how Shot show comparision errors displaying a command you can copy and paste on your local terminal for every screenshot test failed.
 
+If you want to see only failing tests in output, please add `onlyFailing` as parameter.
```

Comment:
We should implement this feature as a param of the [extension we created long ago](https://github.com/Karumi/Shot/blob/master/shot/src/main/scala/com/karumi/shot/ShotExtension.scala) as I mentioned in my comments. It is way easier to use.

Reply:
I currently fail to understand how this ShotExtension can be then configured.
It seems for me like this are "just" constants, which cannot be changed during runtime?